install.rdf

```diff
@@ -44,115 +34,19 @@ Some of the icons and images used are from the Fugue Icons Collection made by Yu
     <em:translator>Alexey Sinitsyn (ru)</em:translator>
     <em:translator>Alessandro Menti (it-IT)</em:translator>
     <em:translator>Engin Özkan (tr)</em:translator>
-    <em:homepageURL>http://www.1st-setup.nl/wordpress/?page_id=133</em:homepageURL>
-    <em:iconURL>chrome://exchangecalendar-common/skin/images/lightningexchangecalendar.png</em:iconURL>
-    <em:type>2</em:type> <!-- type: extension -->
-    <em:unpack>true</em:unpack>
-
-	<em:localized>
-		<Description>
-			<em:locale>en-US</em:locale>
-			<em:name>Exchange EWS Provider</em:name>
-			<em:description>Allows you to sync calendars, tasks, and contacts with your Microsoft Exchange 2007/2010/2013 EWS server from within Lightning.
-
-You can view, delete, create, and update calendar and task/todo items, and manage "out of office" settings.
-You can read and use contacts and global address list contacts for address autocompletion.
-
-Dutch, French, English, German, Swedish, Japanese, Russian, Italian and Turkish localizations.
-
-Some of the icons and images used are from the Fugue Icons Collection made by Yusuke Kamiyamane (http://p.yusukekamiyamane.com/)</em:description>
-		</Description>
-	</em:localized>
-
     <em:localized>
         <Description>
-            <em:locale>de</em:locale>
-                <em:name>Exchange EWS Provider</em:name>
-                <em:description>Ermöglicht es Ihnen Kalender, Aufgaben und Kontakte mit einem Microsoft Exchange 2007/2010/2013 EWS Server aus Lightning heraus zu synchronisieren.
-
-Sie können diese Kalender- und Aufgabeneinträge betrachten, löschen, erstellen und aktualisieren. Sie können die "Out of Office"-Einstellungen verwalten.
-Sie können Kontakte und globale Adresslisten lesen und für die Autovervollständigung der Adressen nutzen.
-
-Übersetzungen in Niederländisch, Französisch, Deutsch, Schwedisch, Japanisch, Russisch, Italienisch und Türkisch.
-
-Einige der Icons und die Bilder werden aus der Fugue Icons Collection erstellt von Yusuke Kamiyamane (http://p.yusukekamiyamane.com/) genutzt.
-               </em:description>
+            <em:locale>en</em:locale>
+            <em:name>Exchange Calendar</em:name>
+            <em:description>Exchange Calendar provides some Microsoft Exchange interfaces for the Lightning calendar.</em:description>
```

Comment:
I would rather state what it does for the user and not what it provides. The original text wasn't that bad in this regard.
How about:
> Allows you to sync calendars, tasks, and contacts with Microsoft Exchange.

Reply:
Well, indeed it looks better for the user and it fits the Mozilla requirement to be a short line description.